Backpack/Map/Classes/BPKMapAnnotationView.m

```diff
@@ -168,20 +191,18 @@ - (void)prepareForReuse {
     self.hasBeenSelected = false;
 }
 
-- (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *_Nullable)event {
-    CGFloat insetBy = self.annotationHitAreaHeight - self.annotationDotHeight;
-    CGRect hitArea = CGRectInset(self.bounds, insetBy, insetBy);
-    return CGRectContainsPoint(hitArea, point);
-}
-
```

Comment:
Turns out this logic didn't actually work, because the map handles taps and determines what should be selected for itself.